make/test.mk

```diff
@@ -302,20 +302,26 @@ endif
 ###########################################################
 
 .PHONY: prepare-projects
-prepare-projects: create-host-project create-member1 create-member2 create-thirdparty-crds
+prepare-projects: create-host-project create-member1-project create-member2-project create-thirdparty-crds
 
-.PHONY: create-member1
-create-member1:
+.PHONY: create-member1-project
+create-member1-project:
 	@echo "Preparing namespace for member operator: $(MEMBER_NS)..."
-	$(MAKE) create-project PROJECT_NAME=${MEMBER_NS}
+	@-oc new-project ${MEMBER_NS} 1>/dev/null
+	@-oc project ${MEMBER_NS}
+	@echo "adding network policies in $(MEMBER_NS) namespace"
+	@-oc process -p NAMESPACE=$(MEMBER_NS) -f deploy/member-operator-network-policies.yaml | oc apply -f -
```

Comment:
it's just a detail - this network policy allows all ingress, so technically dropping the network policy would have the same effect.
This network policy was introduced because you wanted to combine toolchain-e2e setup with the old legacy sandbox-sre setup via cli (which created some network policies at that time). This use-case is not valid anymore, so I guess that we could drop it, right?

Reply:
after all, we don't have anything like that in sandbox-sre, so dropping `member-operator-network-policies.yaml` would make it closer to production usage.

Reply:
> it's just a detail - this network policy allows all ingress, so technically dropping the network policy would have the same effect.

right, we probably don't need it on OpenShift CI on OCP, indeed

> after all, we don't have anything like that in sandbox-sre, so dropping `member-operator-network-policies.yaml` would make it closer to production usage.

I'm confused, we have https://github.com/codeready-toolchain/sandbox-sre/blob/master/resources/setup/default-network-policies.yaml today and it's applied on prod, isn't it?

Reply:
yes, but only in the host namespace, we don't create any network policies in member namespaces at all.

Reply:
I'm talking about dropping the `member-operator-network-policies.yaml` only, not `host-operator-network-policies.yaml`

Reply:
I wonder: if we need to set this `allow-all-ingresses` policy on staging and prod, then is there any benefit in setting it here too, so we have something similar/consistent? or is it overkill and we should just get rid of them? (I'm fine with both options, to be honest)

Reply:
done in 4aed699

Reply:
cool :+1: 
btw, are you going to try to figure out network policies for members too - I mean as a long-term plan?

Reply:
> cool 👍 btw, are you going to try to figure out network policies for members too - I mean as a long-term plan?

yes, let's track it in https://issues.redhat.com/browse/SANDBOX-585